grammerjam/team-boulders
# POST to MYSQL

- Didn't read bullet points for the ticket so you can also add movies to the db.

- To test:

   - Fetch and checkout into the branch

   - Go to postman

      - Type in your localhost URL with appropriate port + "/api/movies" or "/api/users"

      - Make sure your request is a POST request

      - Make sure you set it to raw data and JSON format

      - For users, include:

        - first_name

        - last_name

        - email

        - username

        - password

      - For movies, include:

        - movie_title

        - release_date

        - genre

        - production_company

        - director

        - rating (as in PG or R, etc.)

      - Send the request

      - Watch the magic happen  

        - You should receive the message: "Movie/User added successfully"

 - In the future I hope to make it more DRY
